fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/Trino2SRFunctionCallTransformer.java

```diff
@@ -148,6 +149,18 @@ private static void registerDateFunctionTransformer() {
                 ImmutableList.of(Expr.class));
     }
 
+    private static void registerStringFunctionTransformer() {
+        // chr -> char
+        registerFunctionTransformer("chr", 1, "char", ImmutableList.of(Expr.class));
+
+        // codepoint -> ascii
+        registerFunctionTransformer("codepoint", 1, "ascii", ImmutableList.of(Expr.class));
+
+        // strpos -> locate
+        registerFunctionTransformer("strpos", 2, new FunctionCallExpr("locate",
```

Comment:
do we need to support trino `strpos(string, substring, instance) → bigint` ?

Reply:
No